solutions/cf-588E.mdx

```diff
@@ -0,0 +1,164 @@
+---
+id: cf-588E
+source: CF
+title: Duff in the Army
+author: Chongtian Ma
+---
+
+## Explanation
+
+Note that for each two nodes $u$ and $v$, the path between them is just $u$ to
+$lca(u,v)$ and $v$ to $lca(u,v)$. Also note the constraint of $a \leq 10$ for
+each query. To answer each query, we only need to store the $10$ people with the
+smallest ids for the path from $u$ to $lca(u,v)$ and $v$ to $lca(u,v)$, and
```

Comment:
this doesn't really make sense lol.

Reply:
what doesn’t?

Reply:
it makes sense to me...is there anything specific that's confusing?

Reply:
nvm